spec/plotting/nyaplot/vector_spec.rb

```diff
@@ -37,7 +34,6 @@
     context type.to_s do
       it 'does not set x axis' do
         expect(plot).to receive(:add).with(type, [11, 22, 33]).ordered
-        expect(plot).to receive(:show).ordered
 
         vector.plot(type: type)
```

Comment:
Fine! Previously I was comparing the class using `be_a`, `eq` , instance of class. Also didn't remove the `vector.plot(type: type)` line, that's why I was getting some `InstanceDouble ` error. Now I have updated the testcases.
Thanks!